### PR TITLE
OP-1114 API server workaround in scripts

### PR DESCRIPTION
**Pull request by mizzioisf**
See https://openhospital.atlassian.net/browse/OP-1114

### CODE REVIEW

File: oh.ps1 [hunk 1]
```diff
@@ -1122,6 +1090,52 @@ $JAVA_ARGS="-client -Xms64m -Xmx1024m -Dsun.java2d.dpiaware=false -Djava.library
...
+###################################################################
+function start_api_server {
+	# check for application configuration files
+	if ( !( Test-Path "$OH_PATH/$OH_DIR/rsc/$API_SETTINGS" -PathType leaf )) {
+		Write-Host "Error: missing $API_SETTINGS settings file. Exiting" -ForeGround Red
+		exit 1;
+	}
+	
+	########## WORKAROUND to kill existing API server process ##################
+	########## TO BE REMOVED IN NEXT RELEASES
```
Comment (dbmalkovsky) [inline oh.ps1:1102]:
```suggestion
	########## TO BE REMOVED IN NEXT RELEASE
```

File: oh.ps1 [hunk 2]
```diff
@@ -1501,6 +1515,19 @@ if ( $INTERACTIVE_MODE -eq "on" ) {
 				Write-Host "Killing java..."	
 				Get-Process java -ErrorAction SilentlyContinue | Stop-Process -PassThru
 			}
+			########## WORKAROUND to kill existing API server process ##################
+			########## TO BE REMOVED IN NEXT RELEASES
```
Comment (dbmalkovsky) [inline oh.ps1:1519]:
```suggestion
			########## TO BE REMOVED IN NEXT RELEASE
```

File: oh.sh [hunk 3]
```diff
@@ -983,6 +953,49 @@ function start_gui {
...
+###################################################################
+function start_api_server {
+	# check for application configuration files
+	if [ ! -f ./$OH_DIR/rsc/$API_SETTINGS ]; then
+		echo "Error: missing $API_SETTINGS settings file. Exiting"
+		exit 1;
+	fi
+	
+	########## WORKAROUND to kill existing API server process ##################
+	########## TO BE REMOVED IN NEXT RELEASES
```
Comment (dbmalkovsky) [inline oh.sh:965]:
```suggestion
	########## TO BE REMOVED IN NEXT RELEASE
```

File: oh.sh [hunk 4]
```diff
@@ -1371,6 +1384,19 @@ function parse_user_input {
 			echo "Killing java..."
 			killall java
 		fi
+		########## WORKAROUND to kill existing API server process ##################
+		########## TO BE REMOVED IN NEXT RELEASES
```
Comment (dbmalkovsky) [inline oh.sh:1388]:
```suggestion
		########## TO BE REMOVED IN NEXT RELEASE
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**